quiz/backend/helpers/gen_username/generate.py

```diff
@@ -0,0 +1,28 @@
+import os
+import random
+
+from typing import List
+
+def generate_username(num_results=5) -> List[str]:
```

Comment:
So what I had in mind was something like this article. Sorry for the misleading comment. 

https://realpython.com/python-type-checking/